src/core/components/checkbox/index.tsx

```diff
@@ -17,18 +17,24 @@ import {
 	errorCheckbox,
 } from "./styles"
 import { Props } from "@guardian/src-helpers"
+import {visuallyHidden as _visuallyHidden} from "@guardian/src-foundations/accessibility";
 
 export {
 	checkboxDefault,
 	checkboxBrand,
 } from "@guardian/src-foundations/themes"
 
+const visuallyHidden = css`
+	${_visuallyHidden}
+`
+
 interface CheckboxGroupProps extends Props {
 	// TODO: this is required when an error is passed to correctly set aria-describedby
 	// We should introduce a way of generating element IDs when they are not explicitly passed
 	id?: string
 	name: string
 	label?: string
+	hideLabel?: boolean
```

Comment:
Can we make this mandatory, with a default value of `false`? Here and in other components

Reply:
Of course, one question just for my understanding: What is the difference/gain? `defaultChecked` from `Checkbox` has no default prop either..

Reply:
The difference is really small, but it's a philosophical point that I try to adhere to: how can we minimise complexity?
If `hideLabel` is a optional, its type is effectively:

```ts
hideLabel: true | false | undefined
```

Whereas if we provide a default value, its type is:

```ts
hideLabel: true | false
```

By providing a stricter type, it reduces the number of possible states the component could be in. This makes our component more predictable. The difference is not very dramatic when you focus on a single prop, but every prop we create adds more state permutations.

If you think this is harmful, misguided or if there's something else I might be missing, please let me know. I'd love to hear additional opinions on this 🙂

Reply:
The difference with defaultChecked is that this is overriding a React built-in attribute. In React, [`defaultChecked` is optional](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7d5565b57a32e6d69bd9b3daa94194effd33f6e0/types/react/index.d.ts#L1748), and I didn't want to mess with the expectation. I hope that makes sense, please let me know if you have any thoughts on this.

Reply:
Thanks for the explanation! Sounds legit :)